docs/windows-requirements.md

```diff
@@ -0,0 +1,23 @@
+# Windows
+
+## Requirements to build native modules
+
+If you want to build native modules, make sure that the following tools are installed on your system.
+
+- [Node 14](https://nodejs.org)
+- [Python 3](https://www.python.org/downloads/)
+- [Strawberry Perl](https://strawberryperl.com/)
```

Comment:
Did you need to get Tcl from somewhere as well, or does that come along with one of these...?

Reply:
If you install perl through `choco` it does not come with `Tcl`, but as far as I know installing strawberry Perl added everything required in my `Path`